### PR TITLE
Accessibility Statement

### PR DESCRIPTION
**Pull request by falconedani**
**Link to issue:** https://labcodes.atlassian.net/browse/DSYS-139
**Link to staging app:** https://docs-accessibility--labstorybook-master.netlify.app/?path=/docs/getting-started-accessibility--page

Added an accessibility section in Getting Started with an overview about what accessibility means for the Design System and which methods we are using for testing.

Also updated the [changelog](https://docs-accessibility--labstorybook-master.netlify.app/?path=/docs/getting-started-changelog--page) with an accessibility column and proposed a new convention for the status with descriptions.

### CODE REVIEW

File: stories/ChangeLog.stories.mdx [hunk 1]
```diff
@@ -12,32 +12,50 @@ import { SimpleTag, TogglableTag, RemovableTag, DropdownTag } from "../labsystem
...
+| **Lists**          | <SimpleTag text="Experimental" color="purple" />      | <SimpleTag text="Not started" />                       | -                                                      | -                                                      |
+| **Data Table**     | <SimpleTag text="Experimental" color="purple" />      | <SimpleTag text="Not started" />                       | -                                                      | -                                                      |
+
+## Status description
+Design: to understand which components are available on Figma's Design System or are mapped on the roadmap.
+* <SimpleTag text="Not started" color="mineral" /> The component is mapped but hasn't started yet.
+* <SimpleTag text="Experimental" color="purple" /> The component is ready to be implemented, but misses usability checks (Figma-wise).
+* <SimpleTag text="Stable" color="teal" /> The component is ready to be implemented and has been tested (Figma-wise).
+
+React: to understand which components are available on Figma's Design System or are mapped on the roadmap.
```
Comment (debscorreia) [inline stories/ChangeLog.stories.mdx:49]: React and Design text are the same. Is that intentional? Can be confusing.

**Review by debscorreia — CHANGES_REQUESTED** [review]
@falconedani , I left some comments here and explained a little better on Jira https://labcodes.atlassian.net/browse/DSYS-139 , but nothing big. Let me know if you need help :)

File: stories/Accessibility.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  <input className="simple-tab" type="radio" name="type-tab" id="tab1" defaultChecked/>
+  <label htmlFor="tab1">Overview</label>
+
+  <input className="simple-tab" type="radio" name="type-tab" id="tab2" />
+  <label htmlFor="tab2">Tests</label>
+
+  <div className="simple-tab__content content1">
+  <p>Labcodes believes in web environments made for all people. Confetti follows the global accessibility standard (WCAG) AA Level and try our best to comply with AAA requirements.</p>
+  <h2> Principles</h2>
+  <p>From <a src="https://www.w3.org/WAI/fundamentals/accessibility-principles/">W3C's Web Accessibility Initiative</a></p>
```
[the remark this answers is not in the document]
Comment (falconedani) [reply stories/Accessibility.stories.mdx:24]: Fixed the src/href mistake, sorry about that.

The Accessibility principles were formulated by W3C, they are an umbrella to cover most accessibility issues.

File: stories/ChangeLog.stories.mdx (hunk 1 above)
Comment (falconedani) [reply stories/ChangeLog.stories.mdx:49]: No, it wasn't. Thanks for noticing. Fixed.

File: stories/Accessibility.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  <li>Operable user interface and navigation</li>
+  <li>Understandable information and user interface</li>
+  <li>Robust content and reliable interpretation</li>
+  </ul>
+
+  <h2>Resources</h2>
+  <ul>
+  <li><a src="https://labcodes-wiki.herokuapp.com/best-practices/accessibility/">Labcodes' Web Accessibility Guide</a></li>
+  <li>World Wide Web Consortium (W3C)‘s <a src="https://www.w3.org/WAI/">Web Accessibility Initiative</a> (WAI)</li>
+  <li><a src="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines</a></li>
```
[the remark this answers is not in the document]
Comment (falconedani) [reply stories/Accessibility.stories.mdx]: I was sure I did it 🤔  It's fixed anyway

File: stories/Accessibility.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  <h2> Principles</h2>
+  <p>From <a src="https://www.w3.org/WAI/fundamentals/accessibility-principles/">W3C's Web Accessibility Initiative</a></p>
+  <p>All users must have access to, be able to use, and understand the same content, no matter what disability they might have or what sort of device they might be using to get at that content.</p>
+
+  <p>All content should be/have:</p>
+  <ul>
+  <li>Perceivable information and user interface</li>
+  <li>Operable user interface and navigation</li>
+  <li>Understandable information and user interface</li>
+  <li>Robust content and reliable interpretation</li>
```
[the remark this answers is not in the document]
Comment (falconedani) [reply stories/Accessibility.stories.mdx]: I was sure I did it 🤔  It's fixed anyway

File: stories/Accessibility.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  </ul>
+
+  <h2>Resources</h2>
+  <ul>
+  <li><a src="https://labcodes-wiki.herokuapp.com/best-practices/accessibility/">Labcodes' Web Accessibility Guide</a></li>
```
[the remark this answers is not in the document]
Comment (falconedani) [reply stories/Accessibility.stories.mdx]: Done

**Review by debscorreia — APPROVED** [review]

**Review by lucianoratamero — APPROVED** [review]
LGTM!

**Pull request merged**